build-logic/src/main/kotlin/org.jetbrains.reflekt.conventions.gradle.kts

```diff
@@ -0,0 +1,39 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
+
+import org.gradle.accessors.dm.LibrariesForLibs
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+val libs = the<LibrariesForLibs>()
+
+plugins {
+  id("org.jetbrains.kotlin.jvm")
+}
+
+val ktTarget = KotlinVersion.KOTLIN_2_0
+
+val ktCompilerArgs = listOf(
+    "-Xskip-prerelease-check",
+    "-Xsuppress-version-warnings",
+    "-Xallow-unstable-dependencies",
+    "-opt-in=org.jetbrains.kotlin.ir.symbols.UnsafeDuringIrConstructionAPI",
```

Comment:
Why do we need all these compiler keys now, considering that everything was fine?

Reply:
Some of these are needed for Kotlin 2.0, in particular the final two. I will check if the first two are needed.